docs/data/material/getting-started/templates/dashboard/components/HighlightedCard.js

```diff
@@ -1,34 +1,34 @@
 import * as React from 'react';
-import { styled } from '@mui/material/styles';
-import MuiCard from '@mui/material/Card';
+
+import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import Button from '@mui/material/Button';
 import Typography from '@mui/material/Typography';
 import ChevronRightRoundedIcon from '@mui/icons-material/ChevronRightRounded';
 import InsightsRoundedIcon from '@mui/icons-material/InsightsRounded';
 
```

Comment:
Is there any technical reason for this change? IMO, the old design was much more appealing as a highlighted card for a "new feature advertisement"

Reply:
Not really, is is mostly a proposal, I just forgot to leave a comment. The design feels a bit more consistent to me with only two card styles - but I'll leave it to you to make the final call.